frontend/benefit/handler/src/hooks/useHandlerReviewActions.ts

```diff
@@ -108,6 +109,16 @@ const useHandlerReviewActions = (
         }))
       : undefined;
 
+    const trainingCompensations = values.trainingCompensations
```

Comment:
`values.trainingCompensations?.map` is shorter. Usually also derived values like mapping could be wrapped with `const trainingCompensations = React.useMemo(() => ...values.trainingCompensations?.map(...),[values.trainingCompensations]);` to avoid re-rendering.

Reply:
I have changed to `values.trainingCompensations?.length > 0`, this is because I don't need to use values from `trainingCompensations`, just need to check if they are available

Reply:
Not sure what you mean... the quesition mark would make it anyway shorter whatever is the map function:
<img width="350" alt="Screenshot 2022-02-28 at 12 35 11 PM" src="https://user-images.githubusercontent.com/7648194/155968637-ce6bbc31-0af8-42e5-803e-2c60630205d5.png">
useMemo might help to avoid unnecessary re-renders.